iceoryx_posh/include/iceoryx_posh/popo/modern_api/base_subscriber.hpp

```diff
@@ -76,14 +76,21 @@ class BaseSubscriber : public Condition
     bool hasNewSamples() const noexcept;
 
     ///
-    /// @brief receive Receive the next sample if available.
-    /// @return
-    /// @details Sample is automatically released when it goes out of scope.
+    /// @brief hasMissedSamples Check if samples have been missed since the last take() call.
```

Comment:
I guess it is since the last `hasMissedSamples()` call. Only when you check with this call the state is reset and not with every take. @mossmaurice right, or?

Reply:
Neither reset on `hasMissedSamples()` nor with `take()`, it just calls `empty` on the `ChunkQueuePopper` and the underlying queue. I guess you mixed it up with `hasLostChunksSinceLastCall()`? Should this be replaced?

Reply:
My bad... `ChunkQueuePopper::hasOverflown()` is called here and the flag is reset with every call.

Reply:
OK so @budrus is correct, will update the doc.